performance-storage-service/pss_project/api/models/rest/metrics/SummaryPerformanceMetrics.py

```diff
@@ -0,0 +1,13 @@
+from pss_project.api.models.rest.metrics.BasePerformanceMetrics import BasePerformanceMetrics
+from pss_project.api.models.rest.metrics.MemoryMetrics import MemoryMetrics, MemoryItemSummary
+
+
+class SummaryPerformanceMetrics(object):
```

Comment:
I think this should implement `BasePerformanceMetrics`.

I like this. At first I was confused but now this class makes a lot of sense.

Reply:
Fixed it! Thx!